### PR TITLE
GSoC: remove outdated project and add new JuliaImages projects

### PR DESCRIPTION
**Pull request by timholy**
This adds new projects for improvements in visualization, bindings for OpenCV, and enhancements to image segmentation. I've flagged some potential mentors but not yet gotten permission from any of them, so here goes: @SimonDanisch, @rsrock, @maxruby, @mronian. Of course I'd be thrilled if anyone else stepped forward (this list is not meant to be exclusive).

### CODE REVIEW

**Comment by shashi** [comment]
Thanks so much for doing this! I was not sure if I should remove the project you did.

**Comment by timholy** [comment]
Thanks @shashi. I didn't want to load you up because I suspect you already have more than enough commitments, but obviously your expertise on GUIs/visualization/algorithms would be most welcome.

Huh. Just realized that I spent a big chunk of yesterday working on adding a [feature-comparison matrix](http://juliaimages.github.io/latest/api_comparison.html) for the express purpose of highlighting holes. Should I add this somehow to the images page?

**Comment by shashi** [comment]
Yes! Looks very useful. It's fine if you want to copy-paste this or link to it from a new project item. Thanks!

**Comment by shashi** [comment]
I plan to add a couple of projects related to GUIs and add myself :)

File: soc/ideas-page.md [hunk 1]
```diff
@@ -24,5 +24,6 @@ We have our project ideas organised roughly into the skillsets required:
 * [Numerics](projects/numerics.html) – Challenges for the hard–core number-cruncher, including linear algebra routines and basic mathematical functions.
 * [Differential Equations](projects/diffeq.html) - Numerical methods for high-performance solving of differential equation models
 * [Tooling](projects/tooling.html) – The projects that make the people that make Julia go, go. Help us make a really amazing IDE!
+* [Images](projects/images.hmtl) – extend Julia's suite of tools for visualization and analysis of images.
```
Comment (MikeInnes) [inline soc/ideas-page.md]: typo.

Thanks for this!

Comment (timholy) [inline soc/ideas-page.md]: Good eye, @MikeInnes!

**Comment by timholy** [comment]
I made a couple more tweaks. I also removed myself as a suggested mentor for the numerics projects. I'm willing to provide advice, but to avoid spreading myself too thin I think it's better to focus.

**Comment by MikeInnes** [comment]
Perfect – thanks again. 

Just so people know, being listed as a potential mentor here is a useful thing for students, but doesn't absolutely commit people to anything. Once we have applications we will collectively review proposals and people will have a choice over what they take on (if anything), so we can be careful about anyone having more load than they can take.

This is obviously something we can clarify more carefully, and perhaps we could use some "mentor docs" somewhere. Just noting it here for now anyhow.

**Pull request merged**